### DISCUSSION TITLE
[@types/imapflow] MessageStructureObject.parameters is not a string

### DISCUSSION THREAD

**Discussion by gabbersepp** [DefinitelyTyped/DefinitelyTyped#69810] [Issues with a @types package]
I create this so we can track that issue in our code. I can fix that type issue on myself or you can do. Just let me know.

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/3234900/752ec17a-df53-48a8-a8ad-6734a34dc69e)

[a comment by an automated account is not shown]

**Comment by jeffreyratton98** [comment]
Feel free to make a pr for this. @gabbersepp
